exercises/space-age/.meta/generator/space_age_case.rb

```diff
@@ -0,0 +1,11 @@
+require 'generator/exercise_case'
+
+class SpaceAgeCase < Generator::ExerciseCase
+  using Generator::Underscore
+
```

Comment:
@Insti; I definitely didn't need that constant, it was left over from when each assertion was unnecessarily testing the age on Earth.